src/helpers/ci.ts

```diff
@@ -85,9 +306,9 @@ export const getCIMetadata = (): Metadata | undefined => {
   }
 
   if (env.JENKINS_URL) {
-    const {BUILD_URL, GIT_COMMIT, GIT_BRANCH, [CI_ENV_TRACE_ID]: traceId, [CI_ENV_PARENT_SPAN_ID]: parentSpanId} = env
```

Comment:
continuation of traces through env vars in Jenkins is no longer supported so this code does nothing now